### PR TITLE
Various editorial and tests changes

### PR DESCRIPTION
**Pull request by ptomato**

### CODE REVIEW

File: spec/abstractops.html [hunk 1]
```diff
@@ -108,17 +108,14 @@ <h1>DurationHandleFractions ( _fHours_, _minutes_, _fMinutes_, _seconds_, _milli
         1. Set _seconds_ to floor(_secs_).
         1. Let _fSeconds_ be _secs_ modulo 1.
         1. If _fSeconds_ is not equal to 0, then
-          1. If any of _milliseconds_, _microseconds_, _nanoseconds_ is not 0, throw a *RangeError* exception.
```
Comment (ljharb) [inline spec/abstractops.html:111]: is this not normative?

Comment (ptomato) [reply spec/abstractops.html:111]: I don't believe so — it's dead code, these checks are already performed in step 2a, and the variables are not changed in the meantime.

File: polyfill/lib/calendar.mjs [hunk 2]
```diff
@@ -221,8 +221,8 @@ impl['iso8601'] = {
     const { month, monthCode, ...original } = fields;
     const { month: newMonth, monthCode: newMonthCode } = additionalFields;
     if (newMonth === undefined && newMonthCode === undefined) {
-      original.month = month;
-      original.monthCode = monthCode;
+      if (month !== undefined) original.month = month;
+      if (monthCode !== undefined) original.monthCode = monthCode;
     }
     return { ...original, ...additionalFields };
```
Comment (Ms2ger) [inline polyfill/lib/calendar.mjs:227]: This still doesn't seem to quite match the spec, but this is better, at least.

**Review by Ms2ger — APPROVED** [review]

File: spec/timezone.html [hunk 3]
```diff
@@ -528,7 +528,7 @@ <h1>FormatTimeZoneOffsetString ( _offsetNanoseconds_ )</h1>
       <emu-alg>
         1. Assert: _offsetNanoseconds_ is an integer.
         1. If _offsetNanoseconds_ ≥ 0, let _sign_ be *"+"*; otherwise, let _sign_ be *"-"*.
-        1. Let _nanoseconds_ be abs(_offsetNanoseconds_) modulo 10<sup>9</sup>.
+        1. Let _nanoseconds_ be _offsetNanoseconds_ modulo 10<sup>9</sup>.
```
[the remark this answers is not in the document]
Comment (ptomato) [reply spec/timezone.html:531]: Agreed, I was thinking the wrong way about the definition of modulo. These two commits should just be removed.

File: polyfill/lib/calendar.mjs (hunk 2 above)
Comment (ptomato) [reply polyfill/lib/calendar.mjs]: I've made a better corresponding version, let me know what you think.

**Review by Ms2ger — APPROVED** [review]
Thanks!

**Pull request merged**